master/internal/grpcutil/auth.go

```diff
@@ -100,13 +100,13 @@ func GetUser(ctx context.Context, d *db.PgDB, extConfig *model.ExternalSessions)
 	switch err {
 	case nil:
 		if !user.Active {
-			return nil, nil, ErrPermissionDenied
+			return nil, nil, "", ErrPermissionDenied
 		}
-		return user, session, nil
+		return user, session, token, nil
```

Comment:
question: do tokens have expiration times?  As in, should a fresh container token be created, maybe with a different expiration time?  I'm thinking that reusing the actual token seems a bit fishy to me, but I can't pretend to know our master-side use system so I'm not sure.